### PR TITLE
Remove colored icons

### PR DESCRIPTION
**Pull request by tlunter**
@AdamEdgett 

Removes the colored icons in favor of a template icon and the late train title.

Title and icon will now follow the dark mode rules.

screenshot 2015-01-18 10 43 45
screenshot 2015-01-18 10 44 00

### CODE REVIEW

File: Amtrak Status/TLAmtrakStatusMenu/TLAmtrakStatusMenu.m [hunk 1]
```diff
@@ -72,34 +79,38 @@ - (void)setTrainData:(NSArray*)trains {
...
-            
+
             NSInteger showTimeInMenu = [[NSUserDefaults standardUserDefaults] integerForKey:@"showTimeInMenu"];
             // Always
             if (showTimeInMenu == 2) {
-                [statusItem setTitle:[train estimated]];
+                [self setTitle:[train estimated]];
+                return;
             // When later than 5 minutes
             } else if (showTimeInMenu == 1 && timeDiff > 300) {
```
Comment (AdamEdgett) [inline Amtrak Status/TLAmtrakStatusMenu/TLAmtrakStatusMenu.m]: Could collapse this with the if above with an or. Might be a bit long though:
`showTimeInMenu == 2 || (showTimeInMenu == 1 && timeDiff > 300)`

File: Amtrak Status/TLAmtrakStatusMenu/TLAmtrakStatusMenu.m [hunk 2]
```diff
@@ -72,34 +79,38 @@ - (void)setTrainData:(NSArray*)trains {
...
-            } else if (timeDiff > 900) {
-                [statusItem setImage:[NSImage imageNamed:@"AmtrakYellow"]];
-            } else {
-                [statusItem setImage:[NSImage imageNamed:@"AmtrakBlack"]];
+                [self setTitle:[train estimated]];
+                return;
             }
-        } else {
-            [statusItem setImage:[NSImage imageNamed:@"AmtrakBlack"]];
-            [statusItem setTitle:@""];
```
Comment (AdamEdgett) [inline Amtrak Status/TLAmtrakStatusMenu/TLAmtrakStatusMenu.m]: Does removing this cause the time to stick when it shouldn't?

**Comment by AdamEdgett** [comment]
:

File: Amtrak Status/TLAmtrakStatusMenu/TLAmtrakStatusMenu.m (hunk 1 above)
Comment (tlunter) [inline Amtrak Status/TLAmtrakStatusMenu/TLAmtrakStatusMenu.m]: Eh, it's not too long. Updated.

**Pull request merged**